delfin/leader_election/distributor/task_distributor.py

```diff
@@ -74,6 +74,16 @@ def __call__(self):
         else:
             LOG.debug("Periodic job distribution completed.")
 
+    def distribute_new_job(self, task_id):
```

Comment:
how about function name 'assign_new_job' ?

Reply:
My thought is that the file is `task_distributor`, so I named this function with `distribute`